### DISCUSSION TITLE
Neovim: Performance problem on Windows

### DISCUSSION THREAD

**Discussion by waynebowie99** [community/community#18509] [Copilot Conversations]
I have noticed pretty bad performance while typing when copilot is trying to retrieve results. I would say there's about a .25-.5 second delay sometimes when typing.  I attached a log file of me typing in a file if that's helpful.

I'm running nvim 0.7 but I've tried the latest nightly and the issue still persists

Let me know if you need me to try out anything or provide anything more. 

[vimlog.txt](https://github.com/github-community/community/files/8892762/vimlog.txt)

**Comment by tpope** [comment]
Can you see if there's anything in `:Copilot log`?
